src/openvic-simulation/modifier/ModifierManager.cpp

```diff
@@ -1,11 +1,34 @@
 #include "ModifierManager.hpp"
+#include <string_view>
+#include <utility>
+#include "openvic-simulation/dataloader/NodeTools.hpp"
```

Comment:
```suggestion
#include "ModifierManager.hpp"

#include <string_view>
#include <utility>

#include "openvic-simulation/dataloader/NodeTools.hpp"
```
Spaces between include groups